acceptance-tests/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "acceptance-tests",
+  "version": "0.1.0",
+  "description": "Acceptance tests for cli",
+  "private": true,
+  "author": "Mike Talley <mtalley@hubspot.com>",
+  "homepage": "https://github.com/HubSpot/hubspot-cli#readme",
+  "license": "ISC",
+  "main": "run-tests",
+  "directories": {
+    "lib": "lib"
+  },
+  "files": [
+    "lib"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/HubSpot/hubspot-cli.git"
+  },
+  "scripts": {
+    "test": "echo \"Error: run tests from root\" && exit 1",
+    "run-tests": "./run-tests"
+  },
+  "bugs": {
+    "url": "https://github.com/HubSpot/hubspot-cli/issues"
+  },
+  "devDependencies": {
+    "concat-stream": "^2.0.0",
+    "cross-spawn": "^7.0.3",
+    "dotenv": "^8.2.0",
+    "eslint": "^7.23.0",
```

Comment:
This should be covered by main `eslint` in the `package.json` at the root of the repo so should be removed.

Reply:
Removed